packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -466,76 +425,115 @@ const creatorBehavior = {
       },
     }),
   /**
-   * @param {MethodContext} context
    * @param {string} name
    */
-  getInvitation: (context, name) =>
-    ephemera.directorParamManager.getInternalParamValue(name),
-  /** @param {MethodContext} context */
-  getLimitedCreatorFacet: context => context.facets.machine,
-  getGovernedApis: () => harden({}),
-  getGovernedApiNames: () => harden({}),
+  getInvitation(name) {
+    return ephemera.directorParamManager.getInternalParamValue(name);
+  },
+  getLimitedCreatorFacet() {
+    return this.facets.machine;
+  },
+  getGovernedApis() {
+    return harden({});
+  },
+  getGovernedApiNames() {
+    return harden({});
+  },
 };
 
 const publicBehavior = {
   /**
-   * @param {MethodContext} context
    * @param {Brand} brandIn
    */
-  getCollateralManager: ({ state }, brandIn) => {
+  getCollateralManager(brandIn) {
+    const { state } = this;
     const { collateralTypes } = state;
     collateralTypes.has(brandIn) ||
       Fail`Not a supported collateral type ${brandIn}`;
     /** @type {VaultManager} */
     return collateralTypes.get(brandIn).getPublicFacet();
   },
-  getMetrics: () => ephemera.metricsSubscription,
-
-  /** @deprecated use getCollateralManager and then makeVaultInvitation instead */
-  makeLoanInvitation: makeVaultInvitation,
-  /** @deprecated use getCollateralManager and then makeVaultInvitation instead */
-  makeVaultInvitation,
-  getRunIssuer: () => ephemera.debtMint.getIssuerRecord().issuer,
+  getMetrics() {
+    return ephemera.metricsSubscription;
+  },
+  /**
+   * @deprecated use getCollateralManager and then makeVaultInvitation instead
+   *
+   * Make a vault in the vaultManager based on the collateral type.
+   *
+   */
+  makeVaultInvitation() {
```

Comment:
my eyes are glazing over a bit; I trust this code for `makeVaultInvitation` was just moved